vignettes/ebola.Rmd

```diff
@@ -0,0 +1,615 @@
+We provide the data in the `epidist` package via `?sierra_leone_ebola_data`.
+
+In analyzing this data, we demonstrate the following features of `epidist`:
+
+1. Fitting district-sex stratified delay distribution estimates.
+2. Fitting models with lognormal and gamma delay distributions.
+3. Post-processing and plotting functionality using the integration of `brms` functionality with the [`tidybayes`](http://mjskay.github.io/tidybayes/) package.
+4. Selecting between fitted models using the [`loo`](https://mc-stan.org/loo/) package.
+
+The packages used in this article are:
+
+```{r load-requirements}
+set.seed(1)
+
+library(epidist)
+library(data.table)
+library(purrr)
+library(ggplot2)
+library(sf)
+library(tidybayes)
+library(modelr)
+```
+
+Note that, for users new to `epidist`, before reading this article we recommend beginning with the vignette "[Getting started with `epidist`](http://epidist.epinowcast.org/articles/epidist.html)".
+
+# Data preparation
+
+We begin by loading the Ebola line list data:
+
+```{r}
+data("sierra_leone_ebola_data")
+```
+
+The data has `r nrow(sierra_leone_ebola_data)` rows, each corresponding to a unique case report `id`.
+The columns of the data are the individuals name (retracted, and hence can be removed), age, sex, the dates of Ebola symptom onset and positive sample, and their district and chiefdom.
+
+```{r}
+sierra_leone_ebola_data <- sierra_leone_ebola_data |>
+  dplyr::select(-name) |>
+  dplyr::mutate(case = as.integer(id), .keep = "unused")
+
+head(sierra_leone_ebola_data)
+
+fraction <- 5
+ndistrict <- length(unique(sierra_leone_ebola_data$district))
+```
+
+Figure \@ref(fig:ebola-outbreak) shows the progression of the epidemic in each district.
+(In this figure, we filter down to every `r fraction`th case in order to avoid overplotting.)
+We can see that the start time and course of the epidemic varies between districts.
+
+(ref:ebola-outbreak) Primary and secondary event times for every `r fraction`th case, over the `r ndistrict` districts of Sierra Leone.
+
+```{r ebola-outbreak, fig.cap="(ref:ebola-outbreak)", fig.height=9}
+sierra_leone_ebola_data |>
+  dplyr::filter(case %% fraction == 0) |>
+  ggplot() +
+  geom_segment(
+    aes(
+      x = date_of_symptom_onset, xend = date_of_sample_tested,
+      y = case, yend = case
+    ),
+    col = "grey"
+  ) +
+  geom_point(aes(x = date_of_symptom_onset, y = case), col = "#56B4E9") +
+  geom_point(aes(x = date_of_sample_tested, y = case), col = "#009E73") +
+  facet_wrap(district ~ ., ncol = 2) +
+  labs(x = "", y = "Case ID") +
+  theme_minimal()
+```
+
+We can use a map to help visualise the outbreak across space.
+To create a map, we first join the Ebola data to shapefiles for the districts of Sierra Leone.
+These shapefiles were obtained from the Database of Global Administrative Areas ([GADM](https://gadm.org/)).
+
+```{r message=FALSE, warning=FALSE}
+sf <- sf::st_read("../inst/gadm41_SLE_shp/gadm41_SLE_2.shp", quiet = TRUE)
```

Comment:
use system.file

Reply:
or download from our github on the fly to save package size

Reply:
[72c239b](https://github.com/epinowcast/epidist/pull/52/commits/72c239b7a9c9a536131de92b6cffc4eab688cb29)